Bluejay.asm

```diff
@@ -3781,7 +3804,23 @@ decode_demag_done:
 	mov	Temp1, A
 	jz	decode_temp_done
 
-	mov	A, #(TEMP_LIMIT-TEMP_LIMIT_STEP)
+	; ******************************************************************
+	; Power rating only applies to BB21 because voltage references behave diferently
+	; depending on an external voltage regulator is used or not.
+	; For BB51 (MCU_TYPE == 2) 1s power rating code path is mandatory
+	; ******************************************************************
+IF MCU_TYPE == 0 OR MCU_TYPE == 1
```

Comment:
I think it would be nice if we could be consistent with the MCU_TYPE checks, in other places we use IF `MCU_TYPE < 2`. I have no preference which version we use, as long as it is the same everywhere.

Reply:
Updated